docs/supported-data-types-in-carbondata.md

```diff
@@ -1,29 +1,36 @@
 <!--
     Licensed to the Apache Software Foundation (ASF) under one or more 
     contributor license agreements.  See the NOTICE file distributed with
-    this work for additional information regarding copyright ownership. 
-    The ASF licenses this file to you under the Apache License, Version 2.0
+    this work for additional information reg you under the Apache License, Version 2.0
     (the "License"); you may not use this file except in compliance with 
     the License.  You may obtain a copy of the License at
 
       http://www.apache.org/licenses/LICENSE-2.0
 
-    Unless required by applicable law or agreed to in writing, software 
+    Unless required by  law or agreed to in writing, software 
     distributed under the License is distributed on an "AS IS" BASIS, 
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and 
+    See the License for the specific language  and 
     limitations under the License.
 -->
 
 #  Data Types
 
+
+
 #### CarbonData supports the following data types:
 
   * Numeric Types
     * SMALLINT
     * INT/INTEGER
     * BIGINT
-    * DOUBLE
+    * DOUBLETREE Hduusywnndbb.
```

Comment:
what is this? Hduusywnndbb.

Reply:
jjxhsnn 'mjjzhzjjn'xnnzjjzjjnhzgdhhsnnznzk&&.&/&/72¥.'zi8kkKkzmnmn¥/&/&&/'nzjjhjxhshsbs